service/project.clj

```diff
@@ -16,31 +16,34 @@
   :scm "https://github.com/pedestal/pedestal"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.10.1"]
+  :dependencies [[org.clojure/clojure "1.11.1"]
 
                  [io.pedestal/pedestal.log "0.5.11-SNAPSHOT"]
                  [io.pedestal/pedestal.interceptor "0.5.11-SNAPSHOT"]
                  [io.pedestal/pedestal.route "0.5.11-SNAPSHOT"]
 
+                 [org.ow2.asm/asm "9.3"]
```

Comment:
Why does it is required?

Reply:
I actually no longer recall.  It was probably either one of two things, both related to transitive deps:

1) needed to be updated for the PR to work
2) there was a CVE 

IIRC, it was the former (1).  I can try to look later.